plugins/keycloak-backend/README.md

```diff
@@ -30,8 +30,8 @@ yarn workspace backend add @janus-idp/backstage-plugin-keycloak-backend
      providers:
        keycloakOrg:
          default:
-           # Remove the `/auth` if using keycloak 17+
-           baseUrl: https://<keycloak_host>/auth
+           # Add `/auth` to the baseUrl if using Keycloak versions older than 17
```

Comment:
Let's remove this comment altogether since it implies we could support Keycloak < 17.  We can safely assume all the older versions are EOL